### ISSUE TITLE
Parsing (?) error in Haml view

### ISSUE THREAD

**Issue by JacobEvelyn** [presidentbeef/brakeman#1196]
### Background

Brakeman version: 4.2.1
Rails version: 5.1.4
Ruby version: 2.3.4

#### Other Error

Running `brakeman` on our codebase resulted in an error that appears to stem from the brakeman codebase rather than `ruby_parser`. It occurs when we do this in a (Haml) view:

```haml
- common_locals = { a: 1, b: "two", c: :three }
= render partial: "foo", locals: { **common_locals, d: 4 }
```

This problem appears to go away when we change this to:

```haml
- common_locals = { a: 1, b: "two", c: :three }
= render partial: "foo", locals: common_locals.merge(d: 4)
```

so it's not a big deal, but I thought I'd report it in case others are seeing the same thing.

Stack trace:

```
Error: undefined method `original_line' for nil:NilClass
Location: /Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:121:in `block in process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:120:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:120:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:30:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:45:in `process_partial'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:20:in `process_render'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:813:in `process_if_branch'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:710:in `block (2 levels) in process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:452:in `scope'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:106:in `scope'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:694:in `block in process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `each_with_index'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:49:in `process_iter'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:49:in `process_safely'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:131:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:30:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:45:in `process_partial'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:20:in `process_render'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:49:in `process_iter'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:49:in `process_iter'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:77:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/template_alias_processor.rb:49:in `process_iter'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:811:in `process_if_branch'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:710:in `block (2 levels) in process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:452:in `scope'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:106:in `scope'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:694:in `block in process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `each_with_index'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:693:in `process_if'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:49:in `process_safely'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:131:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/controller_alias_processor.rb:188:in `process_template'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:51:in `process_action'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/render_helper.rb:12:in `process_render'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/processor_helper.rb:5:in `block in process_all'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp.rb:127:in `block in each_sexp'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp.rb:124:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp.rb:124:in `each_sexp'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/lib/processor_helper.rb:4:in `process_all'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/controller_alias_processor.rb:97:in `block in process_defn'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:364:in `block in meth_env'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bundle/ruby/2.5.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:452:in `scope'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:361:in `meth_env'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/controller_alias_processor.rb:90:in `process_defn'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:75:in `block in process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:112:in `in_context'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/ruby_parser/bm_sexp_processor.rb:71:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:61:in `block in process_default'
(eval):3:in `map!'
(eval):3:in `map!'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/alias_processor.rb:59:in `process_default'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processors/controller_alias_processor.rb:31:in `process_controller'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/processor.rb:51:in `process_controller_alias'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:241:in `block (2 levels) in process_controller_data_flows'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:240:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:240:in `block in process_controller_data_flows'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:302:in `block in track_progress'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:299:in `each'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:299:in `track_progress'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:238:in `process_controller_data_flows'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/scanner.rb:61:in `process'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman.rb:352:in `scan'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman.rb:76:in `run'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/commandline.rb:133:in `run_brakeman'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/commandline.rb:118:in `regular_report'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/commandline.rb:142:in `run_report'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/commandline.rb:35:in `run'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/lib/brakeman/commandline.rb:20:in `start'
/Users/jacobevelyn/.gem/ruby/2.3.4/gems/brakeman-4.2.1/bin/brakeman:8:in `<top (required)>'
/Users/jacobevelyn/.gem/ruby/2.3.4/bin/brakeman:22:in `load'
/Users/jacobevelyn/.gem/ruby/2.3.4/bin/brakeman:22:in `<main>'
```

**Comment by presidentbeef** [comment]
Mmmmm yeah Brakeman assumes Hashes are lists of key-value pairs...which they normally are...

**Comment by presidentbeef** [comment]
This was inadvertently affected by #1300 (the error went away), but it's not actually fixed yet.

**Comment by presidentbeef** [comment]
Addressed with #1315 1315
